File: 第 357 场周赛/1.cpp
class Solution {
public:
    string finalString(string s) {
        string res == "";
        for(int i = 0; i < s.length(); i++){
            if(s[i] == 'i'){
                reverse(res.begin(), res.end());
                i++;
            }
            else
                res.push_back(s[i]);
        }
        return res;
    }
};

File: Elementary_algo/23_8_5/1.py
class Solution {
public:
    uint32_t reverseBits(uint32_t n) {
        uint32_t res = 0;
        int i = 0;
        while(i < 32){
            res = res * 2 + n % 2;
            n /= 2;
            i++;
        }
        return res;
    }
};